index.js

```diff
@@ -305,9 +308,7 @@ class CaverExtKAS extends Caver {
      */
     initKIP17API(chainId, accessKeyId, secretAccessKey, url = productionEndpoints.kip17, ver = 'v1') {
         if (url.endsWith('/')) url = url.slice(0, url.length - 1)
-        if (ver === 'v2') {
-            this.kas.initKIP17API(chainId, accessKeyId, secretAccessKey, url, 'v2')
-        } else this.kas.initKIP17API(chainId, accessKeyId, secretAccessKey, url)
+        this.kas.initKIP17API(chainId, accessKeyId, secretAccessKey, url, ver)
     }
```

Comment:
If version 1 is called, please show deprecation log to recommend to update the version.